### PR TITLE
Remove 2.7 support

### PR DESCRIPTION
**Pull request by bsarabeswaran**
Removes python 2.7 from tox and ci

### CODE REVIEW

**Review by jpichon — CHANGES_REQUESTED** [review]
This separate PR should only have the Python 2.7 changes, not both python 2.7 removal and the contents of PR#23.

File: .mergify.yml [hunk 1]
```diff
@@ -6,8 +6,6 @@ queue_rules:
       # Conditions to get out of the queue (= merged)
       - check-success=flake8
       - check-success=integration
-      - check-success=test (2.7)
-      - check-success=test (3.6)
```
Comment (jpichon) [inline .mergify.yml:10]: It looks like 3.6 may have accidentally be removed here?

**Review by jpichon — COMMENTED** [review]
Thank you for the clean-up! I think a line may have accidentally been removed, happy to merge after that!

File: .mergify.yml (hunk 1 above)
Comment (fuzzball81) [reply .mergify.yml:10]: It also looks like 3.10 should be added.

Comment (jpichon) [reply .mergify.yml:10]: I think it's already there after expanding

**Review by jpichon — APPROVED** [review]
Thanks!

**Review by fuzzball81 — APPROVED** [review]

**Pull request merged**